spec/requests/api/v1/conversations/index_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "rails_helper"
+
+describe "GET api/v1/conversations", type: :request do
+
+  let(:topic) { create(:topic) } 
+
+  let(:user_a) { create(:user) }
+
+  let!(:target_1) { create(:target, user: user_a, topic: topic, longitude: 114.2219923, latitude: 22.3129115, radius: 5) }
+
+  let(:user) { create(:user) }
+
+  let(:target_params) do {
+    target: {
+      title: "Meet pop lovers",
+      longitude: 114.2219993,
+      latitude: 22.3129125,
+      radius: 5,
+      topic_id: topic.id
+    }    
+  }
+  end
+
+  before do
+    post api_v1_targets_path, params: target_params, headers: auth_headers, as: :json 
+  end
```

Comment:
What you are testing is getting conversations... so there is no need to create this using the controller. Try to use factories to create the test context

Reply:
Ok, I have done some changes